app/components/samples/table_component.rb

```diff
@@ -85,5 +85,9 @@ def columns
       columns += %i[created_at updated_at attachments_updated_at]
       columns
     end
+
+    def trim_search_phrase(term)
```

Comment:
Doesn't look like this is being used anywhere

Reply:
Thanks.  Removed in de27b8b801ddd560c60efe1d42d5ded8fd7518d2